src/common/serviceUtils.js

```diff
@@ -87,25 +86,32 @@ function handleApiMessages(messages) {
 export function handleRequestError(error) {
   // Validation errors
   if (error.response && error.response.data.field_errors) {
-    logInfo('Field Errors', error.response.data.field_errors);
+    logInfo('API field errors returned to client in field_errors field', { fieldErrors: error.response.data.field_errors });
     handleFieldErrors(error.response.data.field_errors);
```

Comment:
Note for future refactor: this series of handleXXXXXErrors that throw and error may be better named as `throwFieldError` or something so it's clearer that we won't be logging the default case at the end.

Reply:
Agreed - I totally got confused by that when I was reading it.